src/plugins/Sim.js

```diff
@@ -0,0 +1,305 @@
+'use strict';
+
+const OverlayPlugin = require('./util/OverlayPlugin');
+
+/**
+ * Instance sim plugin.
+ * Provides Sim and operator informations control.
+ */
+module.exports = class Sim extends OverlayPlugin {
+    /**
+     * Plugin initialization.
+     *
+     * @param {Object}  instance        Associated instance.
+     * @param {Object}  i18n            Translations keys for the UI.
+     * @param {boolean} basebandEnabled Whether or not baseband control is enabled.
+     */
+    constructor(instance, i18n, basebandEnabled) {
+        super(instance);
+
+        // Reference instance
+        this.instance = instance;
+
+        // Register plugin
+        this.instance.sim = this;
```

Comment:
Looks like there's a cyclical dependency. Sim has a reference to instance, and instance has a reference to sim.

Not sure if there's anyway to avoid this?

Reply:
All player plugins are already setup this way, not sure it's an issue.

Reply:
Ok! :+1: